cannot compile EssentialsX

Hi

I cannot compile Essentials on my own server.
I just doing "git clone https://github.com/EssentialsX/Essentials.git", then "cd Essentials". After that I typed in "mvn clean install".
That gave me the following error:
[ERROR] The build could not read 6 projects -> [Help 1]
[ERROR]
[ERROR]   The project net.ess3:NMSProvider:[unknown-version] (/root/test/Essentials/nms/NMSProvider/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for net.ess3:NMSProvider:[unknown-version]: Failure to find net.ess3:EssentialsXParent:pom:2.0.1 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 5, column 13 -> [Help 2]
[ERROR]
[ERROR]   The project net.ess3:UpdatedMetaProvider:[unknown-version] (/root/test/Essentials/nms/UpdatedMetaProvider/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for net.ess3:UpdatedMetaProvider:[unknown-version]: Failure to find net.ess3:EssentialsXParent:pom:2.0.1 in https://hub.spigotmc.org/nexus/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of spigot-repo has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 5, column 13 -> [Help 2]
[ERROR]
[ERROR]   The project net.ess3:1_8_R1Provider:[unknown-version] (/root/test/Essentials/nms/1_8_R1Provider/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for net.ess3:1_8_R1Provider:[unknown-version]: Failure to find net.ess3:EssentialsXParent:pom:2.0.1 in file:${project.basedir}/lib was cached in the local repository, resolution will not be reattempted until the update interval of project.local has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 5, column 13 -> [Help 2]
[ERROR]
[ERROR]   The project net.ess3:1_8_R2Provider:[unknown-version] (/root/test/Essentials/nms/1_8_R2Provider/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for net.ess3:1_8_R2Provider:[unknown-version]: Failure to find net.ess3:EssentialsXParent:pom:2.0.1 in file:${project.basedir}/lib was cached in the local repository, resolution will not be reattempted until the update interval of project.local has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 5, column 13 -> [Help 2]
[ERROR]
[ERROR]   The project net.ess3:LegacyProvider:[unknown-version] (/root/test/Essentials/nms/LegacyProvider/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for net.ess3:LegacyProvider:[unknown-version]: Failure to find net.ess3:EssentialsXParent:pom:2.0.1 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 5, column 13 -> [Help 2]
[ERROR]
[ERROR]   The project net.ess3:ReflectionProvider:[unknown-version] (/root/test/Essentials/nms/ReflectionProvider/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for net.ess3:ReflectionProvider:[unknown-version]: Failure to find net.ess3:EssentialsXParent:pom:2.0.1 in https://hub.spigotmc.org/nexus/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of spigot-repo has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 5, column 13 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException

It seems that the nms packages are using the wrong version von EssentialsXParent.

--- a/Essentials/pom.xml
+++ b/Essentials/pom.xml
@@ -83,19 +83,19 @@
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>NMSProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>UpdatedMetaProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>1_8_R1Provider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
             <scope>compile</scope>
             <exclusions>
                 <exclusion>
@@ -107,7 +107,7 @@
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>1_8_R2Provider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
             <scope>compile</scope>
             <exclusions>
                 <exclusion>
@@ -119,13 +119,13 @@
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>LegacyProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>ReflectionProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
             <scope>compile</scope>
             <exclusions>
                 <exclusion>

--- a/nms/1_8_R1Provider/pom.xml
+++ b/nms/1_8_R1Provider/pom.xml
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>EssentialsXParent</artifactId>
         <groupId>net.ess3</groupId>
-        <version>2.0.1</version>
+        <version>2.15.0</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
@@ -30,7 +30,7 @@
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>NMSProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
         </dependency>
     </dependencies>
 </project>

--- a/nms/1_8_R2Provider/pom.xml
+++ b/nms/1_8_R2Provider/pom.xml
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>EssentialsXParent</artifactId>
         <groupId>net.ess3</groupId>
-        <version>2.0.1</version>
+        <version>2.15.0</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
@@ -30,7 +30,7 @@
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>NMSProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
         </dependency>
     </dependencies>
 </project>

--- a/nms/LegacyProvider/pom.xml
+++ b/nms/LegacyProvider/pom.xml
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>EssentialsXParent</artifactId>
         <groupId>net.ess3</groupId>
-        <version>2.0.1</version>
+        <version>2.15.0</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
@@ -16,7 +16,7 @@
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>NMSProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
         </dependency>
     </dependencies>
 </project>

--- a/nms/NMSProvider/pom.xml
+++ b/nms/NMSProvider/pom.xml
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>EssentialsXParent</artifactId>
         <groupId>net.ess3</groupId>
-        <version>2.0.1</version>
+        <version>2.15.0</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>

--- a/nms/ReflectionProvider/pom.xml
+++ b/nms/ReflectionProvider/pom.xml
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>EssentialsXParent</artifactId>
         <groupId>net.ess3</groupId>
-        <version>2.0.1</version>
+        <version>2.15.0</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
@@ -23,7 +23,7 @@
          <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>NMSProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
         </dependency>
     </dependencies>
 </project>

--- a/nms/UpdatedMetaProvider/pom.xml
+++ b/nms/UpdatedMetaProvider/pom.xml
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>EssentialsXParent</artifactId>
         <groupId>net.ess3</groupId>
-        <version>2.0.1</version>
+        <version>2.15.0</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>net.ess3</groupId>
             <artifactId>NMSProvider</artifactId>
-            <version>2.0.1</version>
+            <version>2.15.0</version>
         </dependency>
     </dependencies>
 </project>

